src/Nethermind/Nethermind.Merge.Plugin/IMergeConfig.cs

```diff
@@ -64,8 +64,8 @@ The number of requests to the garbage collector (GC) to release the process memo
             """, DefaultValue = "25")]
     public int CollectionsPerDecommit { get; set; }
 
-    [ConfigItem(Description = "The timeout, in seconds, for the `engine_newPayload` method.", DefaultValue = "7", HiddenFromDocs = true)]
-    public double NewPayloadTimeout { get; }
+    [ConfigItem(Description = "The timeout, in miliseconds, for the `engine_newPayload` method.", DefaultValue = "7000", HiddenFromDocs = true)]
+    public int NewPayloadTimeoutMs { get; }
```

Comment:
I'm not sure the "Ms" suffix is needed.

Reply:
This is highly unlikely that this flag is commonly used but we need to know the consequences:
1. Without rename if someone forced it to for example "15" - after this fix it will be 15ms  applied automatically
2. With rename node won't start as this flag does not exist.

Honestly preferring to change the name of flag so it will stop the node rather than automatically start time outing all the payloads.

Other option - keep legacy NewPayloadTimeout and add NewPayloadTimeoutMs as a separate flag and handle the case on what if both are set (pick higher for example and throw warning OR crash at initialization when both are set)

Reply:
I added this flag like last week. So very unused.

Reply:
I think in this case we can assume this flag is not being used

Reply:
However, the consequences are more confusing with option 1, so it's better not to start the node. I changed the flag name to NewPayloadBlockProcessingTimeout.